plugins/modules/iam_user_info.py

```diff
@@ -19,20 +19,25 @@
   name:
     description:
       - The name of the IAM user to look for.
+      - C(user_name) was added as an alias in release 7.2.0.
     required: false
     type: str
+    aliases: ["user_name"]
   group:
     description:
       - The group name name of the IAM user to look for. Mutually exclusive with C(path).
+      - C(group_name) was added as an alias in release 7.2.0.
     required: false
     type: str
-  path:
+    aliases: ["group_name"]
+  path_prefix:
```

Comment:
why not keep `path` as it was before and add aliases?

Reply:
2 of the 3 info modules that supported "path", use "path_prefix", because the API call uses "PathPrefix" (it's a subtree search rather than an exact match search)

Reply:
ok got it